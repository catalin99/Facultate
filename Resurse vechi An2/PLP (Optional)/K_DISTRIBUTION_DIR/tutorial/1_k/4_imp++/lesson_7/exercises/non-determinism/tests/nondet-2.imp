int x;
print(++x,++x,++x);
