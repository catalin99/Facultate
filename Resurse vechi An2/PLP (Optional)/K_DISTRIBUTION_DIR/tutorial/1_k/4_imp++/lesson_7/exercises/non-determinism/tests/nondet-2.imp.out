Search results:

Solution 1:
K:K -->
.K
S:String -->
"123"

Solution 2:
K:K -->
.K
S:String -->
"132"

Solution 3:
K:K -->
.K
S:String -->
"213"

Solution 4:
K:K -->
.K
S:String -->
"231"

Solution 5:
K:K -->
.K
S:String -->
"312"

Solution 6:
K:K -->
.K
S:String -->
"321"
